### PR TITLE
CI: Switch to miniconda setup, install fsl tools through conda

### PR DESCRIPTION
**Pull request by effigies**
Seeking a longer-term solution than #310.

### CODE REVIEW

**Comment by effigies** [comment]
@mgxd @oesteban Any objection? Should we pin to specific versions of FSL tools? We could add a bleeding-edge test in addition to release-pinned tests, so we can report breakage upstream.

**Review by oesteban — APPROVED** [review]
LGTM!

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -34,15 +34,15 @@ jobs:
     strategy:
       max-parallel: 5
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9]
```
Comment (mgxd) [inline .github/workflows/unittests.yml:37]: worth adding 3.10?
```suggestion
        python-version: ["3.8", "3.9", "3.10"]
```

**Review by mgxd — APPROVED** [review]
lgtm

**Comment by mgxd** [comment]
ah, too slow!

**Comment by oesteban** [comment]
Does the fsl channel have a candidate for 3.10? That could be a future culprit

**Comment by effigies** [comment]
The FSL tools we install shouldn't interact with Python versions. We can test on 3.10 in a new PR.

**Pull request merged**